src-js/app.js

```diff
@@ -1,3 +1,77 @@
 import * as Cookie from 'js-cookie';
+import * as lodash from 'lodash.throttle';
 
 window.Cookie = Cookie;
+
+// Instant search initialization
+var search = instantsearch({
+  appId: '06F2EYGKB6',
+  apiKey: 'e80a0ac1e290d3b17ff1ea1692b94285', // search-only API Key; safe for front-end code
+  indexName: 'documentation',
+  routing: true
+});
+
+// initialize SearchBox
+search.addWidget(
+  instantsearch.widgets.searchBox({
+    container: '#search-box',
+    cssClasses: {
+      input: 'instantsearch-search'
+    },
+    placeholder: 'Search Documentation',
+    queryHook: lodash(function (query, searchFunction) { searchFunction(query); }, 1000)
```

Comment:
I think we can now use arrow functions since this file will be transpiled, so this could be written as 
```js
  queryHook: lodash((query, searchFunction) => searchFunction(query), 1000)
```

Reply:
It's probably worth documenting that we're adding this only as a way to do throttling 👍